arbisoft/arbisoft/spiders/jobs_spider.py

```diff
@@ -0,0 +1,75 @@
+import scrapy
+import datetime
+from ..items import JobItem
+import re
+
+
+class JobsSpider(scrapy.Spider):
+    name = 'jobs'
+    start_urls = [
+        'https://news.ycombinator.com/jobs'
+    ]
+
+    def parse(self, response):
+
+        items = response.css("tr.athing td.title, td.subtext")
+        job_titles = []
+        company_urls = []
+        job_urls = []
+        job_posting_dates = []
+        ycombinatorlink = 'https://news.ycombinator.com/'
+        for item in items:
+
+            job_title = item.css("a.storylink::text").get()
+            company_url = item.css("span.sitebit.comhead a span.sitestr::text").get()
+            job_url = item.css("a.storylink ::attr(href)").get()
+            job_posting_date = item.css("span.age a::text").get()
+            if job_posting_date:
+                if 'days' in job_posting_date and int(job_posting_date[:2]) > 5:
+                    break
+            if job_posting_date:
```

Comment:
Why check for `job_posting_date` again if you've already checked for it once?

Reply:
to avoid scrapy crawling additional pages if not required

Reply:
Check on 32 is not needed.

Reply:
done